pandas/tests/arrays/test_timedeltas.py

```diff
@@ -187,6 +188,19 @@ def test_reductions_empty(self, name, skipna):
         result = getattr(arr, name)(skipna=skipna)
         assert result is pd.NaT
 
+    @pytest.mark.parametrize("skipna", [True, False])
+    def test_sum_empty(self, skipna):
+        tdi = pd.TimedeltaIndex([])
+        arr = tdi.array
+
+        result = tdi.sum(skipna=skipna)
```

Comment:
we should have a `min_count` argument if we are going to change this (I know its a PITA but if we do this then its yet another different as EA already has the min_count argument.

Reply:
yah, planning on handling this in a dedicated pass.  these reductions are kind of a mess; ive already got four not-yet-pushed reduction branches on deck.